lib/vanagon/platform/windows.rb

```diff
@@ -50,22 +53,73 @@ def generate_packaging_artifacts(workdir, name, binding)
       # Method to generate the files required to build an MSI  package for the project
       #
       # @param workdir [String] working directory to stage the evaluated templates in
-      # @param name [String] name of the project
+      # @param project [Vanagon::Project] project to build a windows package of
+      # @param binding [Binding] binding to use in evaluating the packaging templates
+      public
```

Comment:
If you use the `private` keyword in bare form, the way you have here, everything after it becomes private. That's why I said the form `private :some_method` is preferred as it only marks the arguments passed as private without affecting any scope defined after it in.